components/background_hang_monitor/sampler_mac.rs

```diff
@@ -112,6 +121,9 @@ unsafe fn frame_pointer_stack_walk(regs: Registers) -> NativeStack {
         if let Err(()) = native_stack.process_register(*pc, *stack) {
             break;
         }
+        if (next <= current) && (next as usize & 3 != 0) {
```

Comment:
This should be ||.

Reply:
Thanks, during development I actually had those into two separate `if` with print statements to see which one would hit, and when I collapsed them into one at the end I off-course made a mistake!